.buildkite/pipeline.yml

```diff
@@ -0,0 +1,27 @@
+steps:
+  - label: Upload pipelines
+    command:
+      - buildkite-agent pipeline upload --replace .buildkite/pipeline.yml
+      - buildkite-agent pipeline upload --replace .buildkite/pipeline.release.yml
+
+  - wait
+
+  - label: ':nodejs: Run JavaScript checks'
+    plugins:
+      - docker#v3.5.0:
+          image: "node:12"
+          workdir: /app
+    command:
+      - npm install --prefix src/yukon/frontend
+      - npm run test --prefix src/yukon/frontend
+      - npm run lint --prefix src/yukon/frontend
+
+  - label: ':python: Run Pyhton checks'
+    command: tox -p auto -e "{py37,py38}-{test,doctest},pylint,mypy,report"
+
+  - wait
+
+  - label: "🚀Release"
+    trigger: "yukon-release"
+    branches: master
+    if: build.tag != null
```

Comment:
nice. The problem is a PR can change the value of "branches:" to trigger a release. This is a security hole we cannot allow.